neuraxle/hyperparams/distributions.py

```diff
@@ -372,7 +380,8 @@ def cdf(self, x) -> float:
         try:
             index = self.choice_list.index(x)
         except ValueError:
-            raise ValueError("Item not find in list. Make sure the item is in the choice list and a correct method  __eq__ is defined for all item in the list.")
+            raise ValueError(
```

Comment:
I just realized this is duplicate code. Please make Choice inherit from PriorityChoice, or PriorityChoice inherit from Choice, I think it will simplify.